index.html

```diff
@@ -12,13 +12,6 @@
       <div class="logo-section">
         <a href="/" class="logo">Rose</a>
       </div>
```

Comment:
- [ ] Just as the previous reviewer pointed out, I would recommend you give it a padding-top so that it would shift down or you align center of the mother tag that caps the logo and the hamburger together.
![image](https://user-images.githubusercontent.com/86200356/220892525-2322e47d-a756-4752-be56-9639af8b2b0f.png)